selftests/unit/test_tree.py

```diff
@@ -176,6 +176,9 @@ def test_advanced_yaml(self):
                          oldroot.children[1].children[2].value)
         self.assertEqual({'new_value': 'something'},
                          oldroot.children[3].children[0].children[0].value)
+        # Convert values, but not keys
```

Comment:
I agree with the approach of converting values (and not keys), but since this differs from the YAML default, we should document it.  Maybe under the "Keys and Values" section on `MultiplexConfig.rst`.